MonstieWash/Assets/Scripts/UIManager.cs

```diff
@@ -1,28 +1,42 @@
 using System;
 using System.Collections;
 using System.Collections.Generic;
+using System.Linq;
 using TMPro;
 using UnityEngine;
 using UnityEngine.UI;
 
 public class UIManager : MonoBehaviour
 {
-    [SerializeField] private TextMeshProUGUI text;
+    public GameObject m_TaskContainer;
+    public GameObject m_TaskTextPrefab;
+
     [SerializeField] private Image clipboard;
     [SerializeField] private Animator CBAnimator;
-
-    private Dictionary<string, float> m_tasks;
+    [SerializeField] private float fontSize = 36f;
+    [SerializeField] private float fontScaling = 0.75f;
+    [SerializeField] private float paddingScaling = 1.2f;
+    [SerializeField] private List<string> m_TaskKeys = new();
```

Comment:
Same here, no need for the prefix.
```suggestion
    [SerializeField] private List<string> taskKeys = new();
```